src/Abc.Zebus/IBus.cs

```diff
@@ -23,6 +23,7 @@ public interface IBus : IDisposable
         IDisposable Subscribe(Subscription subscription, Action<IMessage> handler);
 
         void Reply(int errorCode);
+        void Reply(int errorCode, string message);
```

Comment:
Sort of not backwards compatible and could need a major version, but let's assume there are not too many implementations of IBus.